apps/src/applab/sanitizeHtml.js

```diff
@@ -51,10 +51,16 @@ function warnAboutUnsafeHtml(warn, unsafe, safe, warnings) {
   var ignoredAttributes = [
     'pmbx_context',   // Used by Chrome plugins such as Bitdefender Wallet.
     'kl_vkbd_parsed', // Origin unknown. Assumed to be a plugin of some kind.
+    'kl_virtual_keyboard_secure_input', // Origin unknown.
```

Comment:
I suspect these two `kl_` keys come from the [Kapersky Labs Password Manager](http://www.kaspersky.com/password-manager) - virtual keyboard being their utility for typing passwords while bypassing keyloggers.